scripts/npm-release.js

```diff
@@ -114,7 +114,7 @@ const injectVersions = (packagesToRelease, versions, packages) => {
 
     if (packageJson.dependencies) {
       for (const dependency in packageJson.dependencies) {
```

Comment:
This should be using for...of, right?

Reply:
For...in is for properties and for...of is for iterable objects like arrays - [example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of#Difference_between_for...of_and_for...in)